commit cbd0e6e5d104a3636e8517b3774d7d9dd62d0c89 (refs/changes/34/2334/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-08-08T23:03:26+00:00 (2 years, 2 months ago)
    
    TOOLS-1830 replace sdcadm experimental nfs-volumes with cloudapi-nfs-volumes/docker-nfs-volumes subcommands
    TOOLS-1829 add sdcadm experimental docker-nfs-volumes-automount
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/tools/setup/setup.sh b/tools/setup/setup.sh
index f2f471a..f420625 100644
--- a/tools/setup/setup.sh
+++ b/tools/setup/setup.sh
@@ -317,9 +317,12 @@ upgrade_core_service_to_latest_branch_image "volapi" "tritonnfs"
 # Needed to add additional programs in the HN's GZ, such as sdc-volapi
 upgrade_gz_tools_to_latest_branch_image tritonnfs
 
-# Now enable the experimental_nfs_shared_volumes flag in SAPI
-sdcadm experimental nfs-volumes
-
+# Now enable the flags in SAPI to enable NFS volumes support for docker and
+# CloudAPI, as well as the ability for docker containers to automount NFS
+# volumes
+sdcadm experimental nfs-volumes docker
+sdcadm experimental nfs-volumes docker-automount
+sdcadm experimental nfs-volumes cloudapi
 EOS
 
 echo "VOLAPI setup done, reboot of all CNs is needed before it can be used"
\ No newline at end of file
